src/components/reviews/SingleReview.jsx

```diff
@@ -88,31 +77,24 @@ function SingleReview(props) {
     );
   };
 
-  // handles show/hide button and functionality for body over 250 char
   const bodyOverflow = () => (
     <div>
       {showMore
         ? <ReviewBody>{review.body}</ReviewBody>
         : <ReviewBody>{review.body.slice(0, 251)}</ReviewBody>}
-      <ShowMoreButton
-        onClick={
-          (evt) => {
-            evt.preventDefault();
-            setShowMore(!showMore);
-          }
-        }
-      >
+      <ShowMoreButton onClick={(evt) => setShowMore(!showMore)}>
         {showMore ? 'Show Less' : 'Show More'}
       </ShowMoreButton>
     </div>
   );
 
-  const helpfulClick = () => {
-    markHelpful(review.review_id)
-      .then(() => {
-        setHelpful(helpful + 1);
-      });
-  };
+  const helpfulClick = () => !clicked && markHelpful(review.review_id)
+    .then(() => {
+      setHelpful(helpful + 1);
+      setClicked(true);
+    });
+
+  const unhelpfulClick = () => !clicked && setUnhelpful(unhelpful + 1) && setClicked(true);
 
```

Comment:
So, I believe your problem with clicking the yes or no and it not allowing to then click the other is here because you share the clicked state with both the unhelpfulClick and the helpfulClick functions. So if you click the no it changes the 'clicked' state to true and if you try to click the yes, your helpfulClick function reads the click state as true and does not execute. I think a good way to deal with this is to  is to set the state of clicked to an object with the keys helpful and unhelpful and their values both false. Then you can set state based upon which one was clicked and read state of either depending on which function you execute. You could also just initialize a second state if you'd like, but I believe that is the problem here.

Reply:
I apologize, I didn't explain what I was trying to do very clearly. My goal was to only allow the user to choose yes or no (not both). So if no has been chosen, the user should not be able to also click on yes. Right now if no is clicked, the user can still click yes after. But if yes is clicked first, no can't be clicked (which is what I want to happen)

Reply:
Ah! my misunderstanding, sorry. Then it might be on line 97 you might have to wrap the return in parenthesis. But as it is, it works for now so I will merge it in!